README.md

```diff
@@ -883,8 +883,7 @@ Other Style Guides
     ```javascript
     // bad
     [1, 2, 3].map(number => {
-      const nextNumber = number + 1;
```

Comment:
The only change I think is good in this particular one is the added trailing comma.